doc/api_conf.rst

```diff
@@ -216,6 +216,7 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
     nodocs                  RPMTRANS_FLAG_NODOCS
     justdb                  RPMTRANS_FLAG_JUSTDB
     nocontexts              RPMTRANS_FLAG_NOCONTEXTS
+    nocaps                  RPMTRANS_FLAG_NOCAPS
```

Comment:
@lkundrak Please can you add comment here that it is available only for rpm-4.14 or later. Then we can merge it.